File: M2/12.- Javascript/4.- menu/assets/js/menu.js
let opcion;

// for(inicio; condición; incremento){
//     //ejecución del código
// }

do{
    //1° Capturar lo ingrese el usuario
    opcion = prompt(
        "MEMÚ PRINCIPAL\n" +
        "1. Saludar\n" +
        "2. Mostrar fecha\n" +
        "3. Salir\n\n" +
        "Elige una opción:"
    )

    switch(opcion){
        case "1":
            alert("Hola! Bienvenido al programa");
            break;
        case "2":
            let fecha = new Date();
            alert("Fecha actual: " + fecha.toLocaleDateString());
            break;
        case "3":
            alert("Saliendo del programa...");
            break;
        default:
            alert("Opción no válida, intentelo nuevamente");
    }
    //ejecución del código
}while(opcion !== "3");

File: M3/2.- Sintanxis Python/menu.py
opcion = ""

while opcion !="3":
    #*Mostrar opciones al usuario
    print("\nMenu")#\n salto de línea
    print("1) Saludar")
    print("2) Mostrar Fecha")
    print("3) Salir")

    opcion = input("Elige una opción: ")

    match opcion:
        case "1":
            print("Hola usuario")
        case "2":
            print("Mostrar Fecha")
        case "3":
            print("Saliendo.....")
        case _:
            print("Opción no válida")